src/LightPanels/LightPanels.js

```diff
@@ -862,6 +867,13 @@ module.exports = kind(
 		for (var idx = 0; idx < viewProps.length; idx++) {
 			this.removeTask(this.getViewId(viewProps[idx]));
 		}
+	},
+
+	/**
+	* @private
+	*/
+	setupFallback: function () {
+		this._fallbackTimeout = global.setTimeout(this.bindSafely('transitionFinished'), this.duration + 100);
```

Comment:
Wondering if the extra 100ms is necessary here. Do we care if the fallback fires a moment before the `ontransitionend` event? Is there a risk the transition might not be done in `this.duration` ms but will be in `this.duration + 100`ms and that the user would notice?